oshi-core/src/main/java/oshi/software/os/linux/LinuxOperatingSystem.java

```diff
@@ -191,9 +191,14 @@ public FileSystem getFileSystem() {
     public OSProcess[] getProcesses(int limit, ProcessSort sort) {
         List<OSProcess> procs = new ArrayList<>();
         File[] pids = ProcUtil.getPidFiles();
+        
+        LinuxUserInfo userInfo = getUserInfo();
+        Map<String, String> users = userInfo.getUsers();

```

Comment:
Style: name this `usersMap` or something more self-documenting.  Also, is the key necessarily a `String` or an `Integer`?  I guess if we're saving ourselves a parsing step a String is okay...

Reply:
I used String because in OSProcess userId is a String as well.

Reply:
For future expansion of class, use of String is correct because in Windows we don't have an integer, but a SID in form "S-1-5-21-3623811015-3361044348-30300820-1013"